op-dispute-mon/mon/validator.go

```diff
@@ -42,6 +42,7 @@ func (o *outputValidator) CheckRootAgreement(ctx context.Context, l1HeadNum uint
 		// string match as the error comes from the remote server so we can't use Errors.Is sadly.
 		if strings.Contains(err.Error(), "not found") {
 			// Output root doesn't exist, so we must disagree with it.
+			o.log.Warn("Output root not found", "l1HeadNum", l1HeadNum, "l2BlockNum", l2BlockNum, "rootClaim", rootClaim, "err", err)
```

Comment:
This is going to log a warning on every update of dispute mon, all the time.  We post invalid proposals with future block numbers on a schedule so we _always_ hit this case.  Are we sure this is going to be useful?

Reply:
That's true - let me think on it. There must be a better way to silo this so it's accessible but not overwhelming.

Reply:
What's the actual root cause you're trying to diagnose here?  Sounds like maybe you want to know if the reference op-node has fallen behind?

Reply:
Possibly, we can also close this out - it would just be nice for an investigator to have a way to confirm where the output fetches are failing easily.